src/arch/svm/SpokeUtils.ts

```diff
@@ -596,7 +598,7 @@ export function getRelayDataHash(relayData: RelayData, destinationChainId: numbe
     Uint8Array.from(addressEncoder.encode(SvmAddress.from(relayData.exclusiveRelayer, "base16").toV2Address())),
     Uint8Array.from(addressEncoder.encode(SvmAddress.from(relayData.inputToken, "base16").toV2Address())),
     Uint8Array.from(addressEncoder.encode(SvmAddress.from(relayData.outputToken, "base16").toV2Address())),
-    Uint8Array.from(uint64Encoder.encode(BigInt(relayData.inputAmount.toString()))),
+    arrayify(hexZeroPad(hexlify(relayData.inputAmount), 32)),
```

Comment:
Shouldn't this be the same as the `outputAmount` line on 602?

Reply:
`inputAmount` is now a u8 array in the SVM Spoke
https://github.com/across-protocol/contracts/blob/8da0a9013db1f01b30cea125245d3b1ce1f7e3f3/programs/svm-spoke/src/common/relay_data.rs#L10